File: client/src/models/Order.js
export class Order{
    constructor(data){
        this.id = data.id
        this.createdAt = new Date(data.createdAt).toLocaleTimeString()
        this.updatedAt = new Date(data.updatedAt).toLocaleTimeString()
        this.description = data.description
        this.creator = data.creator
        this.creatorId = data.creatorId
    }
}

File: server/Models/Order.cs
namespace fishbowlManual.Models;
public class Order
{
    public int Id { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
    public string Description { get; set; }
    public Account Creator { get; set; }
    public string CreatorId { get; set; }
}